scripts/misc/docker_pull_baseimage.py

```diff
@@ -14,8 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 from runpy import run_module
+import sys
 
 
 if __name__ == '__main__':
-    run_module('ros_buildfarm.scripts.misc.docker_pull_baseimage', run_name='__main__')
+    try:
+        run_module('ros_buildfarm.scripts.misc.docker_pull_baseimage', run_name='__main__')
+    except ImportError:
+        sys.path.insert(
+            0, os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
+        run_module('ros_buildfarm.scripts.misc.docker_pull_baseimage', run_name='__main__')
```

Comment:
If this pattern weren't already in use across many scripts I'd advocate for an explanatory comment like
```
```suggestion
    try:
        run_module('ros_buildfarm.scripts.misc.docker_pull_baseimage', run_name='__main__')
    except ImportError:
        # Add ros_buildfarm module to the PYTHONPATH using this script as the anchor.
        sys.path.insert(
            0, os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
        run_module('ros_buildfarm.scripts.misc.docker_pull_baseimage', run_name='__main__')
```

But I don't think it is worth chasing down and updating each use or adding it on some but not all instances of this technique. But I'll note it all the same.